docs/wiki/deprecated/discovery.md

```diff
@@ -1,4 +1,9 @@
 ### Flex Auto Container Discovery 
+
+> ⚠️ **Notice** ⚠️: this document contains a deprecated functionality that is still
+> supported by New Relic for backwards compatibility. However, we encourage you to
```

Comment:
is it supported or not?

in the above section we say: "but
its use is discouraged and unsupported because New Relic provides more convenient implementations
of such functionalities."

Reply:
Rephrased